Sources/PromiseKit/Promise.swift

```diff
@@ -873,6 +873,8 @@ public extension Promise where Value == Void {
 }
 
 extension Optional where Wrapped == DispatchQueue {
+  @inlinable
+  @inline(__always)
```

Comment:
Why using `inlinable` and `inline`?

Reply:
The way swift has this implemented is a bit confusing. The in-depth discussion can be seen here: https://forums.swift.org/t/when-should-both-inlinable-and-inline-always-be-used/37375/2

Reply:
I've read about it [here](https://swiftrocks.com/understanding-inlinable-in-swift.html) and [here](https://lists.swift.org/pipermail/swift-users/Week-of-Mon-20170227/004886.html) as well. They don't recommend it. It's public API but undocumented.

Reply:
Removed @inlinable.